developer-docs-site/docs/standards/object.md

```diff
@@ -0,0 +1,239 @@
+---
+title: Object
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Object
+The [Object model](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/object.move) allows Move to represent a complex type as a set of resources stored within a single address and offers a rich capability model that allows for fine-grained resource control and ownership management.
+
+In an object model, an NFT or token can place common token data within a Token resource, object data within an ObjectCore resource, and then specialize into additional resources as necessary, for example, a Player object could define a player within a game. The ObjectCore itself stores both the address of the current owner and the appropriate data for creating event streams.
+
+## Comparison with the account resources model
+The existing Aptos data model emphasizes the use of the store ability within Move. Store allows for a struct to exist within any struct that is stored in global storage, as marked by the key ability. As a result, data can live anywhere within any struct and at any address. While this provides great flexibility it has many limitations:
+
+1. Data is not be guaranteed to be accessible, for example, it can be placed within a user-defined resource that may violate expectations for that data, e.g., a creator attempting to burn an NFT put into a user-defined store. This can be confusing to both the users and creators of this data.
+2. Data of differing types can be stored to a single data structure (e.g., map, vector) via any, but for complex data types any incurs additional costs within Move as each access requires deserialization. It also can lead to confusion if API developers expect that a specific any field changes the type it represents.
+3. While resource accounts allow for greater autonomy of data, it does so inefficiently for objects and does not take advantage of resource groups.
+4. Data cannot be recursively composable, Move currently has a restriction on recursive data structures. Furthermore, experience suggests that true recursive data structures can lead to security vulnerabilities.
```

Comment:
```suggestion
4. Data cannot be recursively composable, because Move currently prohibits recursive data structures. Furthermore, experience suggests that true recursive data structures can lead to security vulnerabilities.
```